src/erlfmt_parse.yrl

```diff
@@ -994,7 +994,7 @@ Erlang code.
 
 %% XXX. To be refined.
 -type error_description() :: term().
--type error_info() :: {erl_anno:line(), module(), error_description()}.
+-type error_info() :: {erl_anno:location() | erlfmt_scan:anno(), module(), error_description()}.
```

Comment:
`ret_err/2` throws a proper `erl_anno:location()`, not sure if it can be returned by `parse_node` or if it always returns `erlfmt_scan:anno()`